content/updates/automation/4-Enhanced-rule-management.md

```diff
@@ -0,0 +1,32 @@
+# Enhanced rule management
+
+## Quickly duplicate your rules
+::: meta-data type="New" ee-only="true" link-to-doc="../articles/manage-your-rules.html#duplicate-a-rule"
+:::
+
+Sometimes you can have very similar rules with just one different condition. In order to avoid rewriting the whole rule, you can now duplicate it in two clicks! What a time-saver, isn't it?
```

Comment:
peut-être élargir l'exemple en parlant d'une différence simple sur les conditions et/ou les actions ? (pour pas que les users croient qu'on peut modifier uniquement les conditions)

Reply:
```suggestion
Sometimes you can have very similar rules with just one different condition or action. In order to avoid rewriting the whole rule, you can now duplicate it in two clicks! What a time-saver, isn't it?
```